test_runner/src/main/kotlin/ftl/gc/GcStorage.kt

```diff
@@ -93,6 +95,15 @@ object GcStorage {
             logLn("Cannot upload performance metrics ${it.message}")
         }.getOrNull()
 
+    fun uploadMatricesId(args: IArgs, matrixMap: MatrixMap) {
+        if (args.disableResultsUpload) return
+        upload(
+            file = args.getMatrixFilePath(matrixMap).toString(),
+            rootGcsBucket = args.resultsBucket,
+            runGcsPath = args.resultsDir
+        )
+    }
```

Comment:
could we add unit test to it?

Reply:
Actually @jan-gogo and I were chatting about it. We decided against it.

Reply:
yep, but still, we can decide to add if someone decides that unit test for it is important.